lib/conduit/sprint/decorators/activate_port_decorator.rb

```
@@ -2,8 +2,12 @@
 
 module Conduit::Sprint::Decorators
   class ActivatePortDecorator < Base
-    def authorized_by
-      (first_name + ' ' + last_name).strip
+    def port_authorized_by
+      if authorized_by
+        authorized_by
+      elsif first_name && last_name
+        (first_name + ' ' + last_name).strip
+      end
```

If neither authorized_by or first and last name are provided, would it be better to throw an exception here rather than waiting for the carrier to tell us it's not a valid request? Should we check to see if first and last name are not blank?